packages/aws-cdk-lib/aws-opensearchservice/lib/domain.ts

```diff
@@ -687,6 +687,15 @@ export interface DomainProps {
    * @default - false
    */
   readonly suppressLogsResourcePolicy?: boolean;
+
+  /**
+   * Whether to enable or disable cold storage on the domain. You must enable UltraWarm storage to enable cold storage.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-opensearchservice-domain-coldstorageoptions.html
+   *
+   * @default - undefined
```

Comment:
defaulted this to `undefined` so there isn't changes in already deployed templates, but could also make this `false` if we want.